src/firstLetters.js

```diff
@@ -60,7 +60,7 @@ function firstLetters(words = '', eng = false, simplify = false) {
     return '';
   }
 
-  return newWords.split(' ').map(firstLetter).join('').replace(/\|/g, '');
+  return newWords.split(' ').map(firstLetter).join('');
```

Comment:
This is definitely needed for gurmukhi, but since #12 mentions punctuation, I think the original intention was to remove `|` when `eng` is `true`.
So, I'd recommend something like:
```suggestion
let letters = newWords.split(' ').map(firstLetter).join('');
if (eng) {
  return letters.replace(/\|/g, '');
}
return letters;
```
or
```suggestion
let letters = newWords.split(' ').map(firstLetter).join('');
return !eng ? letters : letters..replace(/\|/g, '');
```